viewer/vueapp/src/components/connections/Connections.vue

```diff
@@ -571,6 +634,9 @@ export default {
     '$route.query.length': function (newVal, oldVal) {
       this.cancelAndLoad(true);
     },
+    '$route.query.baselineDate': function (newVal, oldVal) {
```

Comment:
as changing the baseline time frame requires running the new query, it causes a `cancelAndLoad`